docs/sql/statements/copy-from.rst

```diff
@@ -372,6 +372,16 @@ specifies the size of one batch. The provided value must be greater than 0, the
 default value is 10000.
 
 
+.. _sql-copy-from-fail_fast:
+
+``fail_fast``
+'''''''''''''
+
+If enabled, the ``COPY FROM`` operation will abort early after an error. This
+is best effort and due to the distributed execution, it may still continue
+processing records before it aborts.
```

Comment:
```suggestion
A boolean value indicating if the ``COPY FROM`` operation should abort early after an error. This
is best effort and due to the distributed execution, it may continue
processing some records before it aborts.

Defaults to ``false``.
```